src-tauri/src/mod_management/mod.rs

```diff
@@ -90,11 +111,17 @@ fn get_installed_mods(game_install: GameInstall) -> Result<Vec<String>, String>
                 continue;
             }
         };
-        mod_names.push(mod_name);
+        let thunderstore_mod_string =
+            match parse_mod_json_for_thunderstore_mod_string(mod_json_path.clone()) {
+                Ok(thunderstore_mod_string) => Some(thunderstore_mod_string),
+                Err(err) => None,
```

Comment:
```suggestion
                Err(_) => None,
```
Compiler throws a warning with this unused variable.

Reply:
Whoops, no clue how I missed that. Thanks for catching it. Fixed with 24eed887c650e3816460166743ee6b4cb17b09e5

Maybe I should look into showing CI warnings on compiler warnings in the future ^^